docs/configuration/settings.rst

```diff
@@ -0,0 +1,586 @@
+| ``'http://kinto.readthedocs.org'``    | be returned in :ref:`the hello view <api-utilities>`.                    |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.project_name ``'kinto'``      | The project name returned in :ref:`the hello view <api- utilities>`.     |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.project_version ``''``        | The version of the project. Will be returned in :ref:`the hello view     |
+|                                       | <api-utilities>`. By default, this is the major version of Kinto.        |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.version_prefix_redirect_enab  | By default, all endpoints exposed by Kinto are prefixed by a             |
+| led  ``True``                         | :ref:`version number <versioning>`. If this flag is enabled, the server  |
+|                                       | will redirect all requests not matching the supported version to the     |
+|                                       | supported one.                                                           |
++---------------------------------------+--------------------------------------------------------------------------+
+
+Example:
+
+.. code-block:: ini
+
+    cliquet.project_name = project
+    cliquet.project_docs = https://project.rtfd.org/
+    # cliquet.project_version = 1.0
+
+Application profiling
+=====================
+
+It is possible to profile the application while its running. This is especially
+useful when trying to find slowness in the application.
+
++---------------------------------------+--------------------------------------------------------------------------+
+| Setting name                          | What does it do?                                                         |
++=======================================+==========================================================================+
+| cliquet.profiler_enabled ``False``    | If enabled, each request will generate an image file with information to |
+|                                       | profile the application.                                                 |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.profiler_dir ``/tmp``         | The Location where the profiler should output its images.                |
++---------------------------------------+--------------------------------------------------------------------------+
+
+Update the configuration file with the following values:
+
+.. code-block:: ini
+
+    cliquet.profiler_enabled = true
+    cliquet.profiler_dir = /tmp/profiling
+
+Render execution graphs using GraphViz:
+
```

Comment:
These instructions are specific to Debuntu and should be clearly marked as such.